pkg/controller/virtualmachineimage/importer_pod.go

```diff
@@ -94,13 +96,6 @@ func GetImporterPodNameFromVmiName(vmiName string) string {
 }
 
 func newImporterPod(vmi *hc.VirtualMachineImage, scheme *runtime.Scheme) (*corev1.Pod, error) {
```

Comment:
Maybe this function can take `(r *ReconcileVirtualMachineImage)` as a receiver so two parameters are not needed and we can use `getSource()` function to identify VMI's source.